modules/manage/pages/kubernetes/manage-resources.adoc

```diff
@@ -234,11 +234,13 @@ helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --crea
 --
 ======
 
+Setting the `min` and `max` configurations to the same values makes sure that Kubernetes assigns a `Guaranteed` Quality of Service (QoS) class to your Pods. Kubernetes uses QoS classes to decide which Pods to evict from a node that runs out of resources. When a node runs out of resources, Kubernetes evicts Pods with a `Guaranteed` QoS last. For more details about QoS, see the https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Kubernetes documentation^].
```

Comment:
Setting the `min` and `max` configurations to the same values ensures that Kubernetes assigns a `Guaranteed` Quality of Service (QoS) class to your Pods.